From 2914d03f28806d86d2264d1ff4cbb234061c7592 Mon Sep 17 00:00:00 2001
From: Matti Viljanen <matti.viljanen@kapsi.fi>
Date: Thu, 14 Apr 2022 00:06:23 +0300
Subject: [PATCH] Fix launching fish_config on SailfishOS

---
 share/tools/web_config/webconfig.py | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/share/tools/web_config/webconfig.py b/share/tools/web_config/webconfig.py
index c5b5290ab..f6a42ee4e 100755
--- a/share/tools/web_config/webconfig.py
+++ b/share/tools/web_config/webconfig.py
@@ -81,6 +81,16 @@ def is_wsl():
     return False
 
 
+def is_sailfish_os():
+    """ Return whether we are running on Sailfish OS """
+    if "linux" in platform.system().lower() and os.access("/etc/sailfish-release", os.R_OK):
+        with open("/etc/sailfish-release", "r") as f:
+            # Find 'ID=sailfishos'
+            if "sailfishos" in f.read():
+                return True
+    return False
+
+
 def is_termux():
     """ Return whether we are running under the Termux application for Android"""
     return "com.termux" in os.environ["PATH"] and find_executable("termux-open-url")
@@ -1639,6 +1649,8 @@ def runThing():
         subprocess.call(["termux-open-url", url])
     elif is_chromeos_garcon():
         webbrowser.open(url)
+    elif is_sailfish_os():
+        subprocess.call(["xdg-open", url])
     else:
         webbrowser.open(fileurl)
 
-- 
2.35.1

